cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/workflows/build_package.yml

```diff
@@ -1,33 +1,41 @@
-name: Build {{ cookiecutter.project_slug }} Package
+name: Build {{ cookiecutter.project_name }} Package
```

Comment:
Didn't we got rid of the project_name?

Reply:
Apparently not, because it would break elsewise :)

project_name allows spaces